bosh-setup/azuredeploy.json

```diff
@@ -66,6 +66,7 @@
   },
   "variables": {
     "apiVersion": "2015-06-15",
```

Comment:
Thanks for catching that - I've closed #934 as we are going to use literal apiVersions for the samples in this repo.  So you can update as such in accordance with the BP.